taxi/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+from taxi_service.settings import AUTH_USER_MODEL
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    class Meta:
+        verbose_name = "Driver"
```

Comment:
In the checklist was mentioned that these names should start with a lowercase letter

Reply:
Not fixed